src/components/checkbox/checkbox.css

```diff
@@ -0,0 +1,12 @@
+@import '../../styles/colors-v2.css';
```

Comment:
We aren't using any of these CSS variables in this file. Let's remove unused imports.

Reply:
D'oh. I copy/pasted css from another file and forgot to remove the imports.

Part of db5d72ea5b088457688e7001e039823573617265